index.js

```diff
@@ -30,6 +38,49 @@ app.get('/ideas', (req, res) => {
     res.status(200).json(getIdeas());
 });
 
+app.get('/ideas/sorted/donationSum', (req, res) => {
+    res.status(200).json(sortedByDonationSumIdeas());
+});
+
+app.get('/ideas/sorted/pendingStatus', (req, res) => {
+    res.status(200).json(pendingIdeasList());
+});
+
+app.get('/ideas/sorted/approvedStatus', (req, res) => {
+    res.status(200).json(approvedIdeasList());
+});
+
+app.post('/ideas', (req, res) => {
+    res.status(200)
+        .json(
+            createIdeas(
```

Comment:
Can this function throw an error?

Reply:
Added a try.. catch statement. Now the method returns Error.message if function throws one. Tested with Postman